chia/cmds/show.py

```diff
@@ -326,16 +217,6 @@ async def show_async(
 
 
 @click.command("show", short_help="Show node information")
-@click.option(
-    "-p",
-    "--rpc-port",
-    help=(
-        "Set the port where the Full Node is hosting the RPC interface. "
-        "See the rpc_port under full_node in config.yaml"
-    ),
-    type=int,
-    default=None,
-)
```

Comment:
This needs to be left in - it's at least used later on.

Reply:
This was accidently removed, i'm going to re review this whole pr before making any more changes

Reply:
Sorry about the review spam, should be good now